tests/pages/services/ServiceActions-cy.js

```diff
@@ -219,13 +219,14 @@ describe('Service Actions', function () {
         .route({
           method: 'PUT',
           url: /marathon\/v2\/apps\/\/cassandra\-healthy/,
-          response: [],
-          delay: 50
-        });
+          response: []
+        }).as('marathon/apps/cassandra-healthy:scaleService');
+
       cy.get('.modal-footer .button-collection .button-primary')
         .as('primaryButton').click();
-      cy.get('@primaryButton').should('have.class', 'disabled');
```

Comment:
I'm happy to add the check again. I only removed it to reduce the number of checks to narrow down the flaky ones and didn't added it again as we have an extra test to verify that the button is properly disabled.